File: LinkERP/ClientApp/src/app/models/sys/lbs-sys-user-roles.ts
import { BaseEntity } from "../base-entity"
import { LbsSysUser } from './lbs-sys-user';
import { LbsSysRole } from './lbs-sys-role';
export class LBS_SYS_UserRoles extends BaseEntity { 
    LoginID: any;
    RoleId: any;
    LBS_SYS_User: LbsSysUser;
    LBS_SYS_Role: LbsSysRole;
}

File: LinkERP.Entity/SYS/LBS_SYS_UserRoles.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace LinkERP.Entity.SYS
{
    public class LBS_SYS_UserRoles : BaseEntity
    {
        public string LoginID { get; set; }
        public Guid RoleId { get; set; }
        public LBS_SYS_User LBS_SYS_User { get; set; }
        public LBS_SYS_Role LBS_SYS_Role { get; set; }
        public string RoleName { get; set; }
        public string RoleCode { get; set; }
    }
}
